autollm/auto/query_engine.py

```diff
@@ -122,6 +124,7 @@ def from_parameters(
             system_prompt (str): The system prompt to use for the query engine.
             query_wrapper_prompt (str): The query wrapper prompt to use for the query engine.
             enable_cost_calculator (bool): Flag to enable cost calculator logging.
+            embed_model (Union[str, EmbedType]): The embedding model to use for generating embeddings. Defaults to OpenAI.
```

Comment:
we should add example for custom hf model usage as `local:intfloat/multilingual-e5-large`